source/crud/tests/unified/estimatedDocumentCount.yml

```diff
@@ -0,0 +1,63 @@
+description: "estimatedDocumentCount"
+
+schemaVersion: "1.1"
```

Comment:
Does this require anything in schema version 1.1? If not, please use 1.0.

Quoting [Top-level Fields: schemaVersion](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#top-level-fields):

> Test files SHOULD be conservative when specifying a schema version. For example, if the latest schema version is 1.1 but the test file complies with schema version 1.0, the test file should specify 1.0.